File: S2T Operational - After - 9.82/S2T Operational - After - 9.82/app/src/main/java/com/myhindlab/abkat/appointment_confirmation/callstatus/CallingRemarkModel.java
package com.myhindlab.abkat.appointment_confirmation.callstatus;


import java.util.List;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CallingRemarkModel {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("output")
    @Expose
    private List<Output> output;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Output> getOutput() {
        return output;
    }

    public void setOutput(List<Output> output) {
        this.output = output;
    }


    public class Output {

        @SerializedName("CReamrkID")
        @Expose
        private Integer cReamrkID;
        @SerializedName("CallingRemark")
        @Expose
        private String callingRemark;

        public Integer getCReamrkID() {
            return cReamrkID;
        }

        public void setCReamrkID(Integer cReamrkID) {
            this.cReamrkID = cReamrkID;
        }

        public String getCallingRemark() {
            return callingRemark;
        }

        public void setCallingRemark(String callingRemark) {
            this.callingRemark = callingRemark;
        }

    }
}

File: lib/calling_modules/models/calling_remark_model.dart
class CallingRemarkModel {
  String? status;
  String? message;
  List<CallingRemarkOutput>? output;

  CallingRemarkModel({this.status, this.message, this.output});

  CallingRemarkModel.fromJson(Map<String, dynamic> json) {
    status = json['status'];
    message = json['message'];
    if (json['output'] != null) {
      output = <CallingRemarkOutput>[];
      json['output'].forEach((v) {
        output!.add(CallingRemarkOutput.fromJson(v));
      });
    }
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['status'] = status;
    data['message'] = message;
    if (output != null) {
      data['output'] = output!.map((v) => v.toJson()).toList();
    }
    return data;
  }
}

class CallingRemarkOutput {
  int? cReamrkID;
  String? callingRemark;

  CallingRemarkOutput({this.cReamrkID, this.callingRemark});

  CallingRemarkOutput.fromJson(Map<String, dynamic> json) {
    cReamrkID = json['CReamrkID'];
    callingRemark = json['CallingRemark'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['CReamrkID'] = cReamrkID;
    data['CallingRemark'] = callingRemark;
    return data;
  }
}
